dissect/target/helpers/localeutil.py

```diff
@@ -0,0 +1,36 @@
+import defusedxml.ElementTree as ET
+from locale import normalize
+
+
+windows_zones = {
+    child.attrib["other"]: child.attrib["type"]
+    for child in ET.parse("./dissect/target/helpers/data/windowsZones.xml").findall(
+        "./windowsZones/mapTimezones/mapZone"
+    )
+    if child.attrib["territory"] == "001"
+}
+
+
+def normalize_timezone(input: str) -> str:
+    """Returns normalized timezone format per IANA TZ standard.
+
+    Takes a Windows registry TimeZoneKeyName string as input, eg.
+    ``Pacific Standard Time`` and translates it to IANA TZ format ``America/Los_Angeles``.
+    Will return the IANA peferred territory 001 value of the corresponding timezone.
```

Comment:
```suggestion
    Takes a Windows registry ``TimeZoneKeyName`` string as input and translates it to IANA TZ format.
    Will return the IANA preferred territory ``001`` value of the corresponding timezone.
    
    For example, ``Pacific Standard Time`` is translated to ``America/Los_Angeles``.
```
Slightly better readable.